### ISSUE TITLE
[Bug]: Support latest versions of certifi

### ISSUE THREAD

**Issue by JamesHutchison** [BerriAI/litellm#1392]
### What happened?

It looks like `litellm` is pegged to the 2023 version of certifi. That library appears to use the year as the major version. I would consider unpegging it since it doesn't use semantic versioning.

### Relevant log output

```shell
`litellm (1.16.21) depends on certifi (>=2023.7.22,<2024.0.0)`
```

Its worth calling out that pegging minimum versions can also create incompatibilities with other libraries, which is actually how this was brought to my attention.

**Comment by ghost** [comment]
i think we can drop the dependency, it's unused. 

Will push a fix, thanks! @JamesHutchison

**Comment by ghost** [comment]
fix pushed, should be live soon in v`1.16.22` - https://github.com/BerriAI/litellm/commit/ebe752fb61ce0c71d87160012e866ab02483e344
